NitroxClient/GameLogic/InitialSync/PlayerPositionInitialSyncProcessor.cs

```diff
@@ -38,13 +38,18 @@ public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualW
                 {
                     SubRoot root = sub.Get().GetComponent<SubRoot>();
                     // Player position is relative to a subroot if in a subroot
-                    if (root != null && !root.isBase)
-                    {                        
+                    if (root != null)
+                    {
+                        // If player is not swimming
                         Player.main.SetCurrentSub(root);
```

Comment:
This line had to be executen whenever a player is nit in the water (so whenever root != null). But it was in the combined if clause so it was only executen when the player was in a vehicle, so I separated the if statements